adapter/service/chargepoint/service.go

```diff
@@ -64,6 +79,14 @@ type Service interface {
 	SendStateReport(force bool) (bool, error)
 	// SupportedStates returns states that are supported by the chargepoint.
 	SupportedStates() []string
+	// SupportsEnergyManagement returns true if chargepoint supports energy management.
+	SupportsEnergyManagement() bool
```

Comment:
Personally, I am really often guilty when it comes to YAGNI, so it's ironic, but maybe it should be just called SupportsChargingMode?

Reply:
Yeah, can be renamed. Done.